src/main/docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:17.0.17_10-jdk
+FROM eclipse-temurin:25
```

Set a specific version, so we know what version we run:

```suggestion
FROM eclipse-temurin:25.0.1_8-jdk
```

---

Sure @Bert-R, done